src/components/TimeSliceGroup.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { PropTypes, Component } from 'react'
+import SelectableTimeSlice from '../containers/SelectableTimeSlice.jsx'
+import date from '../utils/dates.js'
+
+export default class TimeSliceGroup extends Component {
+  static propTypes = {
+    slices: PropTypes.oneOf([2,3,4,5]),
+    step: PropTypes.number.isRequired,
+    value: PropTypes.instanceOf(Date).isRequired,
+    showlabels: PropTypes.bool,
+    isNow: PropTypes.bool,
+    timeGutterFormat: PropTypes.string,
+    timesliceClassnames: PropTypes.string,
+    culture: PropTypes.string
+  }
+  static defaultProps = {
+    slices: 2,
+    step: 10,
```

Comment:
slices become invisible when you go above 5 per slot.  Makes selection very difficult to see